scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/data/TaskProActiveDataspaces.java

```diff
@@ -66,34 +69,38 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
+import java.util.concurrent.locks.ReentrantLock;
 
 import static com.google.common.base.Throwables.getStackTraceAsString;
 
 
 public class TaskProActiveDataspaces implements TaskDataspaces {
 
-    private static final Logger logger = Logger.getLogger(TaskProActiveDataspaces.class);
+    private static final transient Logger logger = Logger.getLogger(TaskProActiveDataspaces.class);
 
     public static final String PA_NODE_DATASPACE_FILE_TRANSFER_THREAD_POOL_SIZE =
             "pa.node.dataspace.filetransfer.threadpoolsize";
 
     public static final String PA_NODE_DATASPACE_CREATE_FOLDER_HIERARCHY_SEQUENTIALLY =
             "pa.node.dataspace.create_folder_hierarchy_sequentially";
 
-    private DataSpacesFileObject SCRATCH;
-    private DataSpacesFileObject INPUT;
-    private DataSpacesFileObject OUTPUT;
-    private DataSpacesFileObject GLOBAL;
-    private DataSpacesFileObject USER;
+    private transient DataSpacesFileObject SCRATCH;
+    private transient DataSpacesFileObject CACHE;
+    private transient DataSpacesFileObject INPUT;
+    private transient DataSpacesFileObject OUTPUT;
+    private transient DataSpacesFileObject GLOBAL;
+    private transient DataSpacesFileObject USER;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "USER" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00116)